### ISSUE TITLE
count of items on selected target on lookup item

### ISSUE THREAD

**Issue by Aokromes** [TrinityCore/TrinityCore#25730]
**Description:**

how about add the count of items someone owns on lookup item some target / name?

**Expected behaviour:**

return amount of items someone owns.

**Branch(es):**

3.3.5

**TC rev. hash/commit:** 

8b2fd345e7b0 N/R

**Comment by illfated** [comment]
Seems useful to me. If there is only 1 of the item in bags or bank, most items don't show up in item count when just pointing to a link pop-up of the item, it needs to be 2 or more for the count to show in-game. Only when the item is craftable through one of the professions will it show up in the craft item window as available from bags (for 1 item).

**Comment by Aokromes** [comment]
oh adding location of item can be cool for stupid people who tells "i don't have xxx but when i try to get it system tells me i can't have it"

**Comment by illfated** [comment]
Fair point, that looks like a typical Unique item scenario, which often catches some unaware users by surprise.
